test/test_checks.py

```diff
@@ -4,10 +4,22 @@
     get_tokens, get_noqa_lines, get_comma_errors,
 )
 
+
+C812 = 'C812 missing trailing comma in enclosure'
 C813 = 'C813 missing trailing comma in Python 3'
 C814 = 'C814 missing trailing comma in Python 2'
 C815 = 'C815 missing trailing comma in Python 3.5+'
 C816 = 'C816 missing trailing comma in Python 3.6+'
+C817 = 'C817 missing trailing comma in function def'
+C818 = 'C818 missing trailing comma in collection'
+
+C822 = 'C822 missing trailing comma in one element enclosure'
+C823 = 'C823 missing trailing comma in Python 3 in one element enclosure'
+C824 = 'C824 missing trailing comma in Python 2 in one element enclosure'
+C825 = 'C825 missing trailing comma in Python 3.5+ in one element enclosure'
+C826 = 'C826 missing trailing comma in Python 3.6+ in one element enclosure'
+C827 = 'C827 missing trailing comma in funtion def with one argument'
+C828 = 'C828 missing trailing comma in one element collection'
```

Comment:
I only lay claim to C81X, You'll have to change this to C8XX

https://github.com/flake8-commas/flake8-commas/blob/master/setup.py#L38